packages/augur-ui/src/utils/generate-template-hash.ts

```diff
@@ -0,0 +1,10 @@
+import { ethers } from 'ethers';
+import { Template } from 'modules/types';
+
+export function generateTemplateHash(template: Template): string {
+  if (!template) return null;
+  let minTemplate = delete template.example;
+  minTemplate = delete template.hash;
+  const value = JSON.stringify(minTemplate, Object.keys(minTemplate).sort());
+  return ethers.utils.sha256(value);
```

Comment:
Realize this is just a first cut, but ultimately this should just be the hash of:
```
{ regex, longDescription }
```